apps/website/src/pages/ambassadors/index.tsx

```diff
@@ -155,15 +183,15 @@ const AmbassadorGroup: React.FC<{
     <div ref={scroll}>
       <Heading
         level={2}
-        className="mb-4"
+        className="mb-8"
         id={`${type}:${camelToKebab(group)}`}
```

Comment:
Given we're now generating the slugs in `sortByOptions`, is this `camelToKebab` needed -- or rather, should this be in the `sortByOptions` logic for the existing options in there (given that the equivalent `kebabToCamel` was removed below)?

Reply:
Sure! That would probably be better. All group keys except for those in classification are already a single lower-case word I think, should I slug them as anyways to future-proof?

Reply:
Yeah, I think having it in there is probably best to future-proof (I assume past me added it in here for that reason)

Reply:
Also should it be based on the key or the actual name of the enclosure, i.e. should it be `#enclosures:critters` or `enclosures:critter-cave`?

Reply:
I think the key, so it remains backwards compatible